src/components/ligneOfButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+/*import calculate from '../logic/calculate';*/
+export class LigneOfButton extends React.Component {
+    constructor(props){
+      super(props)
+      this.props = props;
+      /*this.clickHandler = this.clickHandler*/
+    }
+    /*clickHandler (e){
+      this.props.handler(e);
+    }*/
```

Comment:
- [ ] Kindly delete the commented code. This makes your code looks messy and unprofessional. Please fix this. 

🔧 👍